build.project

```diff
@@ -30,21 +23,44 @@ if [ -z "$TARGET" ]; then
     exit 1
 fi
 
+
 TARGET_FAMILY=$(echo "$TARGET" | awk -F- '{ print $1 }')
-TARGET_SUBFAMILY="$(echo "$TARGET" | awk -F- '{ print $1"-"$2 }')"
+TARGET_SUBFAMILY="$(echo "$TARGET" | awk -F- '{ print $2 }')"
+TARGET_PROJECT="$(echo "$TARGET" | awk -F- '{ print $3 }')"
+
+
+BUILD_TARGET_FILE="_targets/build.project.${TARGET_FAMILY}-${TARGET_SUBFAMILY}"
+BUILD_PROJECT_FILE="_projects/build.project.${TARGET_FAMILY}-${TARGET_SUBFAMILY}-${TARGET_PROJECT}"
+
+
+if [ "$TARGET" != "$TARGET_FAMILY-$TARGET_SUBFAMILY" ] || [ "$TARGET" != "$TARGET_FAMILY-$TARGET_SUBFAMILY-$TARGET_PROJECT" ]; then    
+	b_list_available_targets "TARGET='$TARGET' is unavailable for this project"
```

Comment:
nitpicking, not a merge blocker: the error message should state that TARGET variable has invalid format, not that it's unavailable (we haven't checked that yet)